instances/fileRepositoryType/seafile.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+    "@id": "https://openminds.ebrains.eu/instances/fileRepositoryType/seafile",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/FileRepositoryType",
+    "name": "Seafile repository",
+    "definition": "Seafile is an open source file sync&share solution designed for high reliability, performance and productivity.",
```

Comment:
Is "sync&share" on purpose without spacing?

Reply:
this was copied verbatim from the Seafile website

Reply:
Then we leave it like this.